test/unit/sanitiser/_geo_reverse.js

```diff
@@ -50,7 +50,7 @@ module.exports.tests.sanitize_boundary_country = function(test, common) {
     // t.equals(clean['boundary.circle.radius'], 12.121212, 'should be set to point.lat')
     t.deepEquals(errorsAndWarnings, {
       errors: [],
-      warnings: ['boundary.circle is currently unsupported']
+      warnings: []
```

Comment:
this test is explicitly checking for a warning that `boundary.circle.radius` is going to be ignore, which it is. the code changes you've made don't (or shouldn't) change that behavior. this test is actually proving that the code is broken.